recipes/log4cplus/all/conanfile.py

```diff
@@ -2,13 +2,15 @@
 from conan.tools import files
 from conans import ConanFile, CMake, tools
 
+required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.35.0"
```

`conan.tools.files.get` available since 1.35.0

Reply:
Thank you!